docs/source-code.md

```diff
@@ -25,6 +25,30 @@ Stale reviews should be automatically dismissed.
 
 Repositories should only allow Squash merging.
 
+### Secret detection
+
+All developers are required to [install detect-secrets](developer-laptop-setup/install-detect-secrets.md)
+when working on open-source FFC git repositories in github.
+
+When setting up a new FFC git repository, it needs to be configured to work with the client-side `detect-secrets` tool:
+
+1. Add the configuration file called `.pre-commit-config.yaml` to the repository root directory with the following contents (to exclude additional files/paths from secret detection update the `exclude:` regex):
+
+```
+- repo: git@github.com:Yelp/detect-secrets
+  rev: v0.13.1
+  hooks:
+  - id: detect-secrets
+    args: ['--baseline', '.secrets.baseline']
+    exclude: ^package-lock.json|^.secrets.baseline
```

Comment:
I don't think `.secrets.baseline` needs to be included in the exclude list, at least I've not needed to and the file hasn't been listed as being scanned. I saw this after running the scan with the verbose option turned on to list all scanned files (`-v`).

Reply:
Done (removed the baseline from an exclude)